File: algorithms/C/Count Equal and Divisible Pairs in an Array/Count Equal and Divisible Pairs in an Array.py
class Solution:
    def countPairs(self, nums: List[int], k: int) -> int:
        n=len(nums)
        c=0
        for i in range(0,n):
            for j in range(i+1,n):
                if nums[i]==nums[j] and ((i*j)%k==0):
                    c+=1
        return c

File: scripts/algorithms/C/Count Equal and Divisible Pairs in an Array/Count Equal and Divisible Pairs in an Array.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var countPairs = function(nums, k) {
    var count = 0;
    for(let i=0; i<nums.length; i++){
        for(let j=i+1; j<nums.length; j++){
            if(nums[i] == nums[j] && (i * j) % k == 0){
                count++;
            }
        }
    }
    return count;
};
